src/release/breaking-changes/tooltip-semantics-order.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Accessibility traversal order of tooltip changed 
+description: >
+  the Tooltip widget's message immedately follows the Tooltip widget's child 
+  during accessibility traversal.
+---
+
+## Summary
+
+During accessibility traversal, `Tooltip.message` is visited immedately after
+`Tooltip.child`.
+
+## Background
+
+The `Tooltip` widget usually wraps an interactive UI component such as a button,
+and shows a help message when long pressed. 
+When the message is visible, assistive technologies should announce it after
+the button.
+
+The `Tooltip` widget originally put`Tooltip.message` on an `OverlayEntry` when 
+long pressed.
+As a result, the generated semantics tree does not put `Tooltip.message` 
+immediately after `Tooltip.child`.
+
+You may see accessibility test failures when a tooltip message is visible.
+This is because this update moved the tooltip message in the semantics tree.
+
+## Migration guide
+
+Update your failing accessibility tests. For the following widget tree:
```

Comment:
Maybe
```
The accessibility test before the migration:
```

Reply:
Changed the wording to emphasize the "test" setting.